diff --git a/node_modules/react-to-print/lib/index.d.ts b/node_modules/react-to-print/lib/index.d.ts
index c6b306a..657f201 100644
--- a/node_modules/react-to-print/lib/index.d.ts
+++ b/node_modules/react-to-print/lib/index.d.ts
@@ -1,5 +1,5 @@
-import * as React from 'react';
-import { ReactInstance } from 'react';
+import React from 'react';
+import type { ReactInstance } from 'react';
 export declare const useReactToPrint: (props: IUseReactToPrint) => () => void;
 export interface IReactToPrintProps {
     /**
diff --git a/node_modules/react-to-print/lib/index.js b/node_modules/react-to-print/lib/index.js
index 9243765..f895bdc 100644
--- a/node_modules/react-to-print/lib/index.js
+++ b/src/index.js
@@ -1,5 +1,5 @@
-import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
 
 function useReactToPrint(props) {
   const {
